open-company/open-company-web
# Dev company currency

card: https://trello.com/c/AHJHZzJX

add a currency picker on top of the page, preselect the currency with the cursor value, pass the currency to all the components that need it.
Where the currency symbol is not present it relies on the currency code (3 chars)

to test:
- `lein clean`
- `lein figwheel`
- load the [page](http://localhost:3449/)
- switch currency
- check that the symbol change real time in finances and compensation
